docs/quick-start.md

```diff
@@ -53,3 +53,78 @@ python manage.py update_vector_indexes
 ```
 
 To skip the prompt, use the `--noinput` flag.
+
+## Using event-stream (WagtailVectorIndexSSEConsumer)
+
+`WagtailVectorIndexSSEConsumer` is an asynchronous HTTP consumer designed for handling Server-Sent Events (SSE) for streaming responses from queries using the vector index in real-time. Using the consumer requires ASGI ([uvicorn](https://pypi.org/project/uvicorn/), [Daphne](https://pypi.org/project/daphne/) etc.) along with [django-channels](https://pypi.org/project/django-channels/).
+
+You can configure channels using the [official guide](https://channels.readthedocs.io/en/stable/topics/channel_layers.html#configuration). At a minimum, install the `channels` package and add it to `INSTALLED_APPS` in your settings file.
```

Comment:
**Question:** I'm imagining that given we're not using channel layers, we don't require Redis to run channels, is that correct? Do we actually need to set up channel layers at all? Is this configuration requirement a bit misleading?

Reply:
Correct, the channels layer isn't a dependency, however, the channels package itself is, channels is required for the AsyncHttpConsumer, although now I'm writing this I've noticed that you're talking about the link, which points to the channels layer. 

Fixed by updating the link.